src/services/alpacaService.ts

```diff
@@ -59,22 +139,7 @@ export const postOrder = async (
     limit_price: price,
   });
 
-  try {
-    const res = await fetch(
-      `${brokerApiBaseUrl}/trading/accounts/${accountId}/orders`,
-      {
-        method: "POST",
-        headers: {
-          Authorization: `Basic ${base64EncodedKeys}`,
-        },
-        body,
-      },
-    );
-    return await handleResult(res);
-  } catch (e) {
-    console.log(e);
-    throw Error(`Unable to post order - ${e as string}`);
-  }
+  return postOrder(body);
```

Comment:
Do not have "type" as argument. Set is hardcoded into the body that gets passed to "postOrder", otherwise this method gives no extra value.